### DISCUSSION TITLE
Rustdesk on 32-bit linux

### DISCUSSION THREAD

**Discussion by bt4ibwem8** [rustdesk/rustdesk#4214] [General]
I can see 32-bit .exe build for Windows but no linux builds for this old architecture.
Is it possible to run Rustdesk on 32-bit linux distros? I am on openSUSE TW...

**Comment by cannondale0815** [comment]
As I had to discover myself, while it is possible to build a 32 bit version of the rustdesk client for Linux, the missing link is the libsciter-gtk.so library, which only exists in 64 bit for Linux. This library is required to run the compiled rustdesk binary.

**Comment by SergeyMy** [comment]
yes it is a problem.....but! current version on flutter :)
libsciter removed from project

PS i was able to build rustdesk even for centos7 but still for x64

**Reply by cannondale0815** [reply]
Thank you -- how do you build the flutter version? I found instructions only for sciter.

**Reply by SergeyMy** [reply]
... set environment variables
```
cd flutter
dart pub global activate ffigen --version 5.0.1
flutter pub get
cargo build --features "flutter,flutter_texture_render,hwcodec" --lib --release
flutter build linux --release
```

**Reply by cannondale0815** [reply]
So I learned from the RustDesk Discord that the Flutter version can't be built for 32 bit unfortunately
